NORDIC Embedded Software/nrf52 Patient Monitoring Device/our_service.c

```diff
@@ -35,43 +33,7 @@ EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  *
  */
  
- /*
-static void on_ble_write_acc1(ble_os_t * p_our_service, ble_evt_t * p_ble_evt)
-{
-    // Decclare buffer variable to hold received data. The data can only be 32 bit long.
-    uint32_t data_buffer; 
-    // Pupulate ble_gatts_value_t structure to hold received data and metadata.
-    ble_gatts_value_t rx_data;
-    rx_data.len = sizeof(uint32_t);
-    rx_data.offset = 0;
-    rx_data.p_value = (uint8_t*)&data_buffer;
-    
-    // Check if write event is performed on our characteristic or the CCCD
-    if(p_ble_evt->evt.gatts_evt.params.write.handle == p_our_service->char_handles_acc1.value_handle)
-    {
-        // Get data
-        sd_ble_gatts_value_get(p_our_service->conn_handle, p_our_service->char_handles_acc1.value_handle, &rx_data);
-        // Print handle and value 
-        printf("Value received on handle %#06x: %#010x\r\n", p_ble_evt->evt.gatts_evt.params.write.handle, data_buffer);
-    }
-    else if(p_ble_evt->evt.gatts_evt.params.write.handle == p_our_service->char_handles_acc1.cccd_handle)
-    {
-        // Get data
-        sd_ble_gatts_value_get(p_our_service->conn_handle, p_our_service->char_handles_acc1.cccd_handle, &rx_data);
-        // Print handle and value 
-        printf("Value received on handle %#06x: %#06x\r\n", p_ble_evt->evt.gatts_evt.params.write.handle, data_buffer);
-        if(data_buffer == 0x0001)
-        {
-            printf("Notification enabled\r\n");
-        }
-        else if(data_buffer == 0x0000)
-        {
-            printf("Notification disabled\r\n");
-        }
-    }
-}
-*/
-
+ 
 // ALREADY_DONE_FOR_YOU: Declaration of a function that will take care of some housekeeping of ble connections related to our service and characteristic
 void ble_our_service_on_ble_evt(ble_os_t * p_our_service, ble_evt_t * p_ble_evt)
 {
```

Comment:
case BLE_GATTS_EVT_WRITE:    consider to remove it, commented lines (placed below)